tasks/main.yml

```diff
@@ -51,6 +51,23 @@
     - netstat-nat
   tags:
     - k8s-worker
+  when: ansible_os_family == "Debian"
+
+- name: Install some network packages
+  yum:
+    state: installed
+    name: "{{item}}"
+  with_items:
+    - ebtables
+    - ethtool
+    - ipset
+    - conntrack
+    - iptables
+    - iptstate
+    - net-tools
+  tags:
+    - k8s-worker
+  when: ansible_os_family == "RedHat"
```

Comment:
Normally I would like to support only Ubuntu LTS releases but if this is the only change needed to work on CentOS 7 I'll accept it.

Reply:
Thanks